code/modules/wiremod/components/atom/remotecam.dm

```diff
@@ -0,0 +1,454 @@
+#define REMOTECAM_RANGE_FAR 7
+#define REMOTECAM_RANGE_NEAR 2
+#define REMOTECAM_ENERGY_USAGE_NEAR 0.003 * STANDARD_CELL_CHARGE //Normal components have 0.001 * STANDARD_CELL_CHARGE, this is expensive to livestream footage
+#define REMOTECAM_ENERGY_USAGE_FAR 0.008 * STANDARD_CELL_CHARGE //Far range vision should be expensive, crank this up 8 times
+#define REMOTECAM_EMP_RESET 90 SECONDS
+
+/**
+ * # Remote Camera Component
+ *
+ * Attaches a camera for surveillance-on-the-go.
+ */
+/obj/item/circuit_component/compare/remotecam
```

Comment:
These don't have any reason to a comparison component.
Don't make them one please

Reply:
This doesn't really address my comment. Why do you want these to emulate the comparison component? It's trying to load a component with things it doesn't need and ultimately provides a potential limitation on the output ports this component can have.

It would be better if the person making a circuit would instead just wire up a `Comparison` component up to an `Is Active` output and run that separately instead.

It does involve 2 components, but that's kinda the point of the `Comparison` component.